src/Lykke.Service.EthereumClassicApi/Startup.cs

```diff
@@ -0,0 +1,232 @@
+                    .RegisterModule<ActorsModule>()
+                    .RegisterModule<BlockchainModule>()
+                    .RegisterModule<RepositoriesModule>()
+                    .RegisterModule<ServicesModule>();
+
+                builder
+                    .Register(ctx => ActorSystemFacadeFactory.Build(_container))
+                    .As<IActorSystemFacade>()
+                    .SingleInstance();
+
+                builder
+                    .Populate(services);
+
+                _container = builder.Build();
+
+
+                _actorSystemFacade = _container
+                    .Resolve<IActorSystemFacade>();
+
+                return new AutofacServiceProvider(_container);
+            }
+            catch (Exception e)
+            {
+                WriteFatalError(e, nameof(ConfigureServices));
+
+                throw;
+            }
+        }
+
+        private async Task CleanUp()
+        {
+            try
+            {
+                await _log.WriteMonitorAsync("", $"Env: {Program.EnvInfo}", "Terminating");
+
+                _container.Dispose();
+            }
+            catch (Exception ex)
+            {
+                WriteFatalError(ex, nameof(CleanUp));
+
+                throw;
+            }
+        }
+
+        private IReloadingManager<AppSettings> LoadSettings()
+        {
+            var configuration = new ConfigurationBuilder()
+                .SetBasePath(_environment.ContentRootPath)
+                .AddEnvironmentVariables()
+                .Build();
+
+            return configuration.LoadSettings<AppSettings>();
+        }
+
+        private static void SetupSwagger(SwaggerOptions options)
+        {
+            options.PreSerializeFilters.Add
+            (
+                (swagger, httpReq) => swagger.Host = httpReq.Host.Value
+            );
+        }
+
+        private static void SetupSwaggerGen(SwaggerGenOptions options)
+        {
+            options.SwaggerDoc("v1", new Info
+            {
+                Title = "Ethereum Classic API",
+                Version = "v1"
+            });
+        }
+
+        private static void SetupSwaggerUI(SwaggerUIOptions options)
+        {
+            options.RoutePrefix = "swagger/ui";
+
+            options.SwaggerEndpoint("/swagger/v1/swagger.json", "v1");
+        }
+
+        private async Task StartApplication()
+        {
+            try
+            {
+                await _log.WriteMonitorAsync("", $"Env: {Program.EnvInfo}", "Started");
+            }
+            catch (Exception ex)
+            {
+                WriteFatalError(ex, nameof(StartApplication));
+
+                throw;
+            }
+        }
+
+        private async Task StopApplication()
+        {
+            try
+            {
+                if (_actorSystemFacade != null)
+                {
+                    await _actorSystemFacade.ShutdownAsync();
+                }
+            }
```

Comment:
Log terminating to the monitor as in the service template

Reply:
It's loggen in the cleanup method, as in template: https://github.com/LykkeCity/lykke.dotnettemplates/blob/0b506d6d6326a558c7107c40c4f78f7ae1f5bac0/Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Startup.cs#L152